packages/segmented-control/src/SegmentedControlOption.tsx

```diff
@@ -0,0 +1,238 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { Size, Mode } from './types';
+
+/**
+ * Styles
+ */
+
+// The border color is slightly different from the base gray for accessibility reasons
+const checkedSelector = '[data-checked="true"]';
+const uncheckedSelector = '[data-checked="false"]';
+
+const optionStyle = ({
+  mode = 'light',
+  size = 'default',
+}: {
+  mode: Mode;
+  size: Size;
+}) => css`
+  ${size === 'small' &&
```

Comment:
Since this is already a function, we don't need to be interpolating this conditional within the return value. You might consider writing the function like this:

```js
const optionStyle = () => {
  if (size === 'small') {
    return css``;
  }
  
  // Etc...
}
```

Reply:
I'll have to think on this. Since the function takes in both `size` and `mode`, the returned css depends on both those parameters. We can't elegantly concatenate the results of two `css\`\`` calls, so I'll have to consider refactoring this a bit.